app/models/status.rb

```diff
@@ -32,6 +32,9 @@ class Status < ApplicationRecord
   include Paginable
   include Cacheable
   include StatusThreadingConcern
+  include RateLimitable
+
+  rate_limit by: :account, family: :statuses
```

Comment:
From what I understand, any scheduled status for an account that reached its limit will error out (in the publish worker). I am not sure this is how it *should* work.